oracle/src/events/processTransfers/index.js

```diff
@@ -55,6 +57,15 @@ function processTransfersBuilder(config) {
           return
         }
 
+        const existsTokensSwappedEvent = receipt.logs.some(e => e.topics[0] === tokensSwappedHash)
+
+        if (from === ZERO_ADDRESS && existsTokensSwappedEvent) {
```

Comment:
Will it be more safe to also check that the emitter of `TokensSwapped` is the bridge contract address?

Reply:
Yes, applied the change https://github.com/poanetwork/tokenbridge/pull/224/commits/d0117edf6bc555d5690b1b90d5596a9b89114f14